en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,12 +1,22 @@
 = MuleSoft Composer Release Notes
-:page-revdate: October 5, 2023
+:page-revdate: October 16, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer] for a list of known issues and limitations by date.
 
+== October 16, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|Implemented backend changes in HubSpot Connector to support Mule Runtime 4.5.0. | W-14224733
```

Comment:
add to SF Composer too?

Reply:
I am not sure - I wouldn't think so since it is for Mule? I sent a message but I won't hear back until tomorrow.